### PR TITLE
v0.154.3

### PR DESCRIPTION
**Pull request by xlgmokha**
## v0.154.3, 19 June 2021

- fix: Fetches upload-pack using git if http fails
- fix(Terraform): handle dependencies without a namespace
- chore: Replace wget with curl for minimized dependency
- chore: add lint job to check shell scripts
- build(deps): bump @npmcli/arborist in /npm_and_yarn/helpers
- chore: Double quote variables in Dockerfile's shellscript, cc #3917

https://github.com/dependabot/dependabot-core/compare/v0.154.2...1b17a40

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,12 @@
+## v0.154.3, 19 June 2021
+
+- fix: Fetches upload-pack using git if http fails
```
Comment (thepwagner) [inline CHANGELOG.md:3]: Question: some of these are community contributions, do those contributors want a shout-out?

I tend to include a `(@awesome-contributor)` for anyone outside GitHub's Dependabot team as means of thanks and encouragement.
It's one of those weird things that's done but not written down anywhere. We could not care, we could write it down, or encode it in the [bump-version](https://github.com/dependabot/dependabot-core/blob/main/bin/bump-version.rb#L37) script

**Review by thepwagner — APPROVED** [review]
LGTM; I wondered about the (unwritten) attribution policy.
Since it's not written, not a blocker. :shipit:

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,12 @@
...
+- fix(Terraform): handle dependencies without a namespace
+- chore: Replace wget with curl for minimized dependency
+- chore: add lint job to check shell scripts
+- build(deps): bump @npmcli/arborist in /npm_and_yarn/helpers
+- chore: Double quote variables in Dockerfile's shellscript, cc #3917
```
Comment (feelepxyz) [inline CHANGELOG.md:8]: Going to copy this to a new release as I merged a few changes without spotting this release

**Comment by feelepxyz** [comment]
https://github.com/dependabot/dependabot-core/pull/3957

File: CHANGELOG.md (hunk 1 above)
Comment (xlgmokha) [reply CHANGELOG.md:3]: That's a great idea.

File: CHANGELOG.md (hunk 2 above)
Comment (xlgmokha) [reply CHANGELOG.md:8]: Thank you for taking care of this @feelepxyz.

**Pull request closed without merging**